battleship-pseudo.txt

```diff
@@ -0,0 +1,21 @@
+// PACE Group 1 - Battleship Pseudocode
+
+// Structs: Game Board - array of chars with defined sizes
+// 	    Ship       - Size and direction/facing to place on board
+
```

Comment:
// Declare variables for ocean, part of ship, part of ship that's been hit,
// and represents a hit